docs/DESIGN_GUIDELINES.md

```diff
@@ -852,7 +852,9 @@ export interface ILogGroup {
 }
 ```
 
-### Imports
+### References
+
+> References create a read-only pointer to the unowned resources
```

Comment:
I think we should also add information about _why_ someone may want to do this, given it is a read-only pointer. @mrgrain thoughts?

Reply:
Resource created outside of the current stack/app. Personally I like to think about this as a very basic way of service discovery. Often you have centrally managed resources that are available under a well-known address. Think an SNS Topic that publish company-wide notifications. There's really no need to have anything more complex than a hardcoded string for these.